contents/tutorials/calendly-webhooks.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to capture events from Calendly webhooks
+date: 2023-09-21
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-14.png
+tags: ['events']
+---
+
+Webhooks enable you to send data from one platform to another when an event happens. This enables you to run whatever workflows and code to handle those events.
+
+To showcase the power of webhooks, we are going to capture Calendly meeting data into PostHog using Val Town, a platform for writing and running JavaScript functions in your browser. 
+
+> **Note:** Calendly webhooks requires their professional plan.
+
+## Set up your Val Town webhook handler
+
+Before we set anything up in Calendly, we set up our webhook handler function in [Val Town](https://val.town/). 
+
+To get started, create a variable file in Val Town with your `phProjectAPIKey`. Get your project API key from your [project settings](https://app.posthog.com/project/settings), go to Val Town, create a function like `let phProjectAPIKey = ‘<ph_project_api_key>’` and press "Run". This saves the value for us to reference as `@me.phProjectAPIKey` later.
+
+![Project API key Val](../images/tutorials/calendly-webhooks/val.png)
+
+Next, we create our function to capture a Calendly webhook. It takes a request, parses the JSON, gets the data we want, and then uses PostHog to capture it.
+
+> Val Town enables you to reference or fork functions other people wrote, like the `capturePostHogEvent` [function](https://www.val.town/v/ianvph.capturePostHogEvent) we can use here.
```

Comment:
Instead of mention this, in your code, can we just add the code from "  @ianvph.capturePostHogEvent(", instead of referencing somewhere else? WIll be easier to follow